core/server/data/schema/default-settings.json

```diff
@@ -34,10 +34,10 @@
             }
         },
         "logo": {
-            "defaultValue": "https://static.ghost.org/v1.0.0/images/ghost-logo.svg"
+            "defaultValue": "https://static.ghost.org/v3.0.0/images/ghost.png"
```

Comment:
Not sure we want to change the name of the file here form `ghost-logo` to `ghost`?